mobile/library/objective-c/EnvoyConfiguration.mm

```diff
@@ -244,4 +249,9 @@ - (instancetype)initWithAdminInterfaceEnabled:(BOOL)adminInterfaceEnabled
   }
 }
 
+- (NSString *)bootstrapDebugString {
```

Comment:
nit proposal: maybe change to `descritpion` to make it follow patterns established by other iOS frameworks?
```suggestion
- (NSString *)bootstrapDebugDescription {
```